orchagent/portsorch.cpp

```diff
@@ -3120,12 +3120,18 @@ void PortsOrch::updateDbPortFlapCount(Port& port, sai_port_oper_status_t pstatus
     std::time_t now_c = std::chrono::system_clock::to_time_t(now);
     if (pstatus == SAI_PORT_OPER_STATUS_DOWN)
     {
-        FieldValueTuple tuple("last_down_time", std::ctime(&now_c));
+        char buffer[32];
+        // Format: Www Mmm dd hh:mm:ss yyyy
+        std::strftime(buffer, sizeof(buffer), "%a %b %d %H:%M:%S %Y", std::localtime(&now_c));
```

Comment:
@arista-nwolfe can we keep the time in UTC since the syslog event time is in UTC?

Reply:
Will do, note that previously it was already in local time with `ctime`